config/web_portal/menu.xml

```diff
@@ -59,29 +59,29 @@
 	  <show_on_instance>write_enabled</show_on_instance>
     </spacer>
     
-    <Add_site>
+    <AddSite>
```

Comment:
what was the logic for making these changes to `menu.xml`

Reply:
That's a good question. It was it impose some consistency so that the values here could be programatically converted to the local_info.xsd schema definition. So, a commit crossover, and needs a bit more investigation to figure out if I've missed something in refactoring to separate commits.

Reply:
So ...
Commit https://github.com/ineilson/gocdb/commit/e9487dff699ef2f9778a32d28ddaa1f05dcbe498 contains .xslt code to  generate the menu element specifications which are included in the .xsd file. The XML schema enforcement/checking is not yet in the dev branch so menu elements (and colour spec.) elements don't exist yet to be validated. After the XML schema commit is merged then the present commit will need updating with the required XML schema elements, as per the commit above.

Reply:
I'm a little confused... is ineilson@e9487df on an older branch where the URL specific views and schema checking were more intertwined?

Reply:
Yes, that sums it up. Whichever branch (this filter-by-scope or local_info-schema-checking) is merged first, the other will need some changes.